exonum-java-binding-common/src/main/java/com/exonum/binding/common/proofs/list/ListProofElement.java

```diff
@@ -56,7 +72,7 @@ public void accept(ListProofVisitor visitor) {
     INSTANCE {
       @Override
       public void funnel(ListProofElement from, PrimitiveSink into) {
-        into.putBytes(from.element);
+        into.putBytes(from.element.toByteArray());
```

Comment:
I'd perhaps keep a funnel here as it abstracts how this element is hashed,
however, there is less sense than there used to be: previously it allowed to avoid a copy incurred when accessing the element from the outside to hash it
(`getElement() -> element.clone()` vs `this.element` in the funnel).